src/classes/PSC_Opportunity_TDTM.cls

```diff
@@ -84,18 +84,20 @@ public with sharing class PSC_Opportunity_TDTM extends TDTM_Runnable {
 
         if (objType == Partial_Soft_Credit__c.SObjectType) {
             //query all PSCs associated with these opps, update their ISO code
-            for (Partial_Soft_Credit__c psc : database.query('SELECT Id, Opportunity__c, CurrencyIsoCode FROM Partial_Soft_Credit__c WHERE Opportunity__c IN :oppIds')) {
-                string oppCurrency = mapOppIdToISO.get(psc.Opportunity__c);
+            for (Partial_Soft_Credit__c psc : Database.query('SELECT Id, Amount__c, Opportunity__c, CurrencyIsoCode FROM Partial_Soft_Credit__c WHERE Opportunity__c IN :oppIds')) {
+                String oppCurrency = mapOppIdToISO.get(psc.Opportunity__c);
 
                 psc.put('CurrencyIsoCode', oppCurrency);
+                psc.put('Amount__c', psc.Amount__c);
```

Comment:
I'm always a bit wary when it comes to using non-namespaced field names in a string. It **may** work in a namespaced org, but it may fail. Given that we know the `Amount__c` field exists here (and below), it might be safer to just use `psc.Amount__c = psc.Amount__c;` (though we'll want to test that it still works as expected). The alternative is to call the UTIL_Namespace class to apply the namespace to the field.